configure.ac

```diff
@@ -272,6 +270,20 @@ else
     AC_MSG_RESULT([no])
 fi
 
+# Control Envoy support
+#
+AC_ARG_ENABLE([envoy],
+    AS_HELP_STRING([--enable-envoy],
```

Comment:
can we configure both nginx and envoy?

Reply:
yep, `--enable-nginx --enbale-envoy` should be accepted